src/extension.ts

```diff
@@ -79,6 +79,7 @@ export async function activate(context: vscode.ExtensionContext) {
 
 		const hostService = new HostService(context, notificationService, logger);
 		context.subscriptions.push(hostService);
+		await hostService.updateSSHRemotePlatform();
```

Comment:
should be done after `didFirstLoad`, extension should activate as soon as possible